VideoLocker/src/main/java/org/edx/mobile/view/MyCourseListTabFragment.java

```diff
@@ -57,6 +57,7 @@ protected void loadData(boolean showProgress) {
         //This Show progress is used to display the progress when a user enrolls in a Course
```

Comment:
nit: Might be a good time to either remove this totally obvious comment or correct its grammar.